src/ResourceManager/AzureBatch/Commands.Batch/Pools/StartBatchPoolResizeCommand.cs

```diff
@@ -29,9 +29,14 @@ public class StartBatchPoolResizeCommand : BatchObjectModelCmdletBase
         [ValidateNotNullOrEmpty]
         public string Id { get; set; }
 
-        [Parameter(Mandatory = true, HelpMessage = "The number of target dedicated compute nodes.")]
+        [Parameter(Mandatory = false, HelpMessage = "The number of target dedicated compute nodes.")]
         [ValidateNotNullOrEmpty]
-        public int TargetDedicated { get; set; }
```

Comment:
needs toi be documented for the migration guide

Reply:
Fixed